packages/muelu/src/Utils/MueLu_AlgebraicPermutationStrategy_def.hpp

```diff
@@ -237,7 +242,7 @@ namespace MueLu {
       if(gColVec->getMap()->isNodeGlobalElement(globColId)) {
         MyWeightForColId[myRank] = gColId2Weight[globColId];
       } else {
-        MyWeightForColId[myRank] = 0.0;
+        MyWeightForColId[myRank] = SC_ZERO;
```

Comment:
Are weights Scalars or magnitudes?

Reply:
@mhoemmen I do not know what the intent is here, I guess @jhux2 and @csiefer2 are the best persons to answer that question.
I simply tried to be consistent using the same type as the array entries.

Reply:
`MyWeightsForColId` should be magnitude type, as after a global reduction, it's compared to `winnerValue`, around line 257.

Reply:
@jhux2 Does that make this code syntactically incorrect?  `Teuchos::ScalarTraits<Scalar>::zero()` should be complex, right?